src/Identifier/Resolver/ResolverInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Authentication\Identifier\Resolver;
+
+interface ResolverInterface
```

Comment:
I agree with @robertpustulka that ResolverInterface etc should be enough

Otherwise

    class OrmResolver implements ResolverInterface

should also be

    class OrmIdentityResolver implements IdentityResolverInterface

to be consistent here. And that is a bit too much maybe.